runtime/src/inflation_rewards/points.rs

```diff
@@ -221,12 +225,18 @@ mod tests {
         }
     }
 
+    fn into_vote_state_view(vote_state: VoteState) -> VoteStateView {
+        let vote_account_data =
+            bincode::serialize(&VoteStateVersions::new_current(vote_state)).unwrap();
+        VoteStateView::try_new(Arc::new(vote_account_data)).unwrap()
+    }
```

Comment:
fn is repeated a couple times, should we just add this to the vote state view module as a `From` impl that is `#cfg(test)`?